extensions/amp-nested-menu/0.1/amp-nested-menu.js

```diff
@@ -100,6 +100,10 @@ export class AmpNestedMenu extends AMP.BaseElement {
         submenuBtn.setAttribute('tabindex', 0);
       }
       submenuBtn.setAttribute('role', 'button');
+      // Initialize aria-expanded for a11y
+      if (submenuBtn.hasAttribute('amp-nested-submenu-open')) {
```

Comment:
Why isn't it designated `aria-expanded="true"` when the submenu is open?

Also you can rewrite as:
`submenuBtn.setAttribute('aria-expanded', !submenuBtn.hasAttribute('amp-nested-submenu-open'));`

But again maybe leave a comment since at a glance this appears unintuitive.

Reply:
Bad naming here, `amp-nested-submenu-open` doesn't indicate that the submenu is open but rather identifies the button that opens the submenu, should be called `amp-nested-submenu-open-btn` instead really...

Reply:
Thanks for clarifying. Is this button intended to be exposed for styling? Please also add a comment, something like `// The menu is closed when the submenu open button is available.`